public/cover.css

```diff
@@ -0,0 +1,176 @@
+/*
+ * Globals
+ */
+
+/* Links */
+a,
+a:focus,
+a:hover {
+  color: #fff;
+}
+.nav .open>a, .nav .open>a:focus, .nav .open>a:hover {
```

Comment:
Even though it is optional, it is common practice (as seen in the [CSS spec examples](https://www.w3.org/TR/2009/PR-css3-selectors-20091215/#combinators)) to add whitespace around combinator selectors. This makes for much more readable selectors:

`.nav .open>a` → `.nav .open > a` 🙂